frontend/src/utils/constants.tsx

```diff
@@ -1,14 +1,12 @@
 /**
- * The base URL determined by the current environment.
- * @returns {string}
+ * @constant {string} API_URL - The base URL determined by the current environment.
```

Comment:
I think this format is a bit more consistent with the other documentation we have so I changed it

Reply:
I'm pretty sure we came up with this tag ourselves 🥳 
here is the link on how to properly parse such tags in our docs: https://github.com/TypeStrong/typedoc/issues/2614#issuecomment-2187850425

Reply:
Ahhh, I see.  Thanks, man 🙏